sklearn/multioutput.py

```diff
@@ -202,8 +200,7 @@ def predict(self, X):
             raise ValueError("The base estimator should implement"
                              " a predict method")
 
-        X = check_array(X, force_all_finite=False, accept_sparse=True)
-
+        X = self._validate_data(X, force_all_finite=False, accept_sparse=True)
```

Comment:
Since this is predict:

```suggestion
        X = self._validate_data(X, force_all_finite=False, accept_sparse=True,  
                                reset=False)
```